YellowPod/YellowPod/Common/Extensions/UIApplication+Ext.swift

```diff
@@ -2,39 +2,22 @@
 //  UIApplication+Ext.swift
 //
 
-import Foundation
 import UIKit
 
+/**
+ Place a top view controller depending on current root controller.
+ */
 extension UIApplication {
     class func topViewController(controller: UIViewController? = UIApplication.shared.keyWindow?.rootViewController) -> UIViewController? {
         if let navigationController = controller as? UINavigationController {
             return topViewController(controller: navigationController.visibleViewController)
         }
-        if let tabController = controller as? UITabBarController {
-            if let selected = tabController.selectedViewController {
-                return topViewController(controller: selected)
-            }
+        if let tabController = controller as? UITabBarController, let selected = tabController.selectedViewController  {
```

Comment:
This line seems too long to me. Would it be a good idea to split the if statement?

Maybe we should set a maximum permitted amount of chars per line.

Reply:
The style guide we are following does not have any rules related. I agree with your comment, maybe we could add this as a "code review recipe" in [our guides repo](https://github.com/yellowme/guides/blob/master/code-review/SWIFT.md)

Reply:
I just reviewed the new styleguide version:

> - Long lines should be wrapped at around 70 characters. **A hard limit is intentionally not specified.**
> - Avoid trailing whitespaces at the ends of lines.
> - Add a single newline character at the end of each file.

Unlike `a multiple guard` statement, the `multiple if let` statement does not looks pretty when you separate each `let` per line.

Reply:
This refactor reduces the line a bit:

```swift
if let selectedFromTabBar = (controller as? UITabBarController)?.selectedViewController {
    return topViewController(controller: selectedFromTabBar)
}
```